recipients_app/lib/view/widgets/income/payment_card.dart

```diff
@@ -17,14 +17,6 @@ class PaymentCard extends StatelessWidget {
     super.key,
   });
 
-  String cleanStatus(PaymentStatus? currentStatus) {
```

Comment:
I would define as a rule, the lifecycle methods (initState, build, dispose) should be first because they define how a widget should look like and that's the first thing we want to know